src/utils/constants/orderSignatures.ts

```diff
@@ -0,0 +1,61 @@
+import * as Abi from 'web3-eth-abi';
+
+function joinSignature(name, argTypes = []) {
+  return `${name}(${argTypes.join(',')})`;
+}
+
+function abiEncode(joinedSignature) {
+  return Abi.encodeFunctionSignature(joinedSignature);
+}
+
+export const makeOrderSignature = joinSignature('makeOrder', [
```

Comment:
Actually I don't understand the value of this. We should derive these signatures directly from the ABI, or just provide strings. Why make an array and join it directly?

Reply:
We need signature string along with bytes. I didn't find web3 function that returns signature string by taking function name as parameter. And joining as a separate step is just for better readability of signature string.